site/zh/blog/new-in-devtools-93/index.md

```diff
@@ -0,0 +1,148 @@
+
+容器查询特性现还处于实验性质。测试该特性之前，请在 `chrome://flags` 那里打开 `#enable-container-queries` 开关。
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/3NzGBpukHQfUZUKUpUgf.png", alt="Editable CSS container queries in the Styles pane", width="800", height="554" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/46cdd9cd019f088e1134abe84dbc7d53ac60585a #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/a7e1eac63bee3728b41ae440f2ec250559e9c667 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/ef157dab2ccf321941548a51d350f9383a78d283 #}
+
+Chromium 议题: [1146422](https://crbug.com/1146422)
+
+
+## 网络面板可预览 Web bundle {: #web-bundle }
+[Web bundle](https://web.dev/web-bundles/) 是一种新兴的文件格式，用于封装一个或多个 HTTP 资源。现在，您可以在**网络**面板那里预览 web bundle的网络请求。
+
+web bundle 特性现还处于实验性质。测试该特性之前，请在 `chrome://flags` 那里打开 `#enable-experimental-web-platform-features` 开关。
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/PEv1mNA14K18t5P3N6Yj.png", alt="web bundle preview", width="800", height="492" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/e7672c40f2febc80786632c188b6029b2f2ac7b7 #}
+
+Chromium 议题: [1182537](https://crbug.com/1182537)
+
+
+## Attribution Reporting API 调试 {: #attribution-reporting }
+现在，Attribution Reporting API 错误会显示在**问题**面板那里。
+
+[Attribution Reporting](https://developer.chrome.com/docs/privacy-sandbox/attribution-reporting/)是一个新的 API，旨在帮助您在没有使用跨站标识符的情况下，评估用户行为（比如点击或者查看广告）所导致的转化率。
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/bkEGVEv5kKc9M6qBUmLz.png", alt="Attribution Reporting API errors in the Issues tab", width="800", height="501" %}
+
+Chromium 议题: [1190735](https://crbug.com/1190735)
+
+
+## 更好地处理控制台里面的字符串 {: #string }
+**控制台**里面的新右键菜单允许您复制任何字符串，作为文本内容、JavaScript 字面量或者 JSON 字面量。
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/O5uMSgkHrQ2mQDSjmg3A.png", alt="New context menu in the Console", width="800", height="477" %}
+
+在 Chrome 90，**控制台**默认将[输出字符串格式化为 JSON 字面量](/blog/new-in-devtools-90/#double-quotes)。我们从开发者那里得到反馈：这个改动让人有点迷，一些人觉得字符串的转义有点多，不利于字符串的可读性。
+
+现在，**控制台**默认将输出字符串格式化为有效的 JavaScript 字面量，并且会进一步提供 3 种复制字符串的选项。**复制字符串作为 JavaScript 字面量**的这个选项，会转义那些合适且特殊的字符，然后会根据字符串内容来决定是否使用单引号、双引号或者反引号来包裹字符串。**复制字符串内容**的这个选项呢，会一字不差地将原生的字符串内容（包括换行符以及其它的特殊字符）复制到剪切板。最后，**复制字符串作为 JSON 字面量**的这个选项，会将字符串格式化为有效的 JSON 字面量，然后复制到剪切板。
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/9242d13569e9fe67ac01e75d28fa2b6e6bf310d2 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/5715a7b9800532d8b28e2c9fa2d3c1e220ba54a8 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/29236e333a856ae5a952fe4182545b1e2bde5539 #}
+
+Chromium 议题: [1208389](https://crbug.com/1208389)
+
+
+## 改善 CORS 调试体验 {: #cors }
+**控制台**里面的 CORS 相关错误，现在可以链接到**网络** 和**问题**面板。
+
+点击 CORS 相关错误信息旁边的两个新按钮以查看网络请求，以及进一步理清楚该错误信息，在**问题**面板那里获取潜在的解决方案。
```

Comment:
I'd suggest using "问题标签页" to translate "Issues tab", since this is consistent with DevTools' localization: https://source.chromium.org/chromium/chromium/src/+/main:third_party/devtools-frontend/src/front_end/core/i18n/locales/zh.json;l=3377;drc=fc02ab2f743adfba1b673402891d8411dd395f1c